tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/alerts/alerts_container.ts

```diff
@@ -58,16 +74,24 @@ export class AlertsContainer {
         (alertsBreakdown) => {
           const alertTypes = Object.keys(alertsBreakdown);
           alertTypes.sort();
-          return alertTypes.map((alertType) => {
-            return {
-              ...ALERT_TYPE_TO_DISPLAY_NAME_AND_SYMBOL[alertType],
-              count: alertsBreakdown[alertType],
-            };
-          });
+          return alertTypes.map(
+            (alertType): AlertTypeDisplay => {
+              return {
+                ...ALERT_TYPE_TO_DISPLAY_NAME_AND_SYMBOL[alertType],
```

Comment:
feels like the `type` could have been populated here instead of changes to L38, L43, and L48. WDYT?

Reply:
I like what you suggested: slightly less code.